Anytype/Sources/PresentationLayer/Modules/AllContent/AllContentSettingsMenu.swift

```diff
@@ -20,8 +21,10 @@ struct AllContentSettingsMenu: View {
             mode
             Divider()
             sortByMenu
-            Divider()
-            bin
+            if !FeatureFlags.showBinWidgetIfNotEmpty {
+                Divider()
+                bin
+            }
```

Comment:
should we delete bin from settings menu? I think it might be a nice addition.

Reply:
yes, maybe, asked Kostya 🙌